### ISSUE TITLE
Add sklearn interface improvement

### ISSUE THREAD

**Issue by jcr-lyxor** [aksnzhy/xlearn#233]
Following [#68](https://github.com/aksnzhy/xlearn/issues/68) that would be very nice to have extra information in the model. Maybe I could find it but I do see that in the documentation:

1. Recovering the early stopping epoch number
2. Recovering the cross validation test and train losses / metrics
3. Recovering eval_set  losses / metrics
4. Be able to pass a path to a libffm data format 

I am happy to contribute even if I have no idea on how to get 1 and 2.

The library is really nice and fast. Thanks.

**Comment by etveritas** [comment]
@jcr-lyxor these features is already in xLearn, like early stoppping. If you want to train a ffm model, you can use the native api, i.e. pass the pass of libffm to mdoel. For sklearn interface,  we are improving.

**Comment by jcr-lyxor** [comment]
What I meant is how do you know the number of epoch the model stopped on? Also I do not see how to output the CV errors or if the CV is performed.

**Comment by etveritas** [comment]
@jcr-lyxor 
If you set the validate file when train, the console prints like this, and I point the information you may need.
![20190313231540](https://user-images.githubusercontent.com/27916175/54292674-928e3300-45e9-11e9-998e-3ced0a7d8763.png)
If you use CV, the console prints like this, and I point the information you may need.
![20190313233505](https://user-images.githubusercontent.com/27916175/54292748-b2bdf200-45e9-11e9-81eb-790f36a8e019.png)
Hope can help you.

**Comment by jcr-lyxor** [comment]
Yes thanks you but how can I recover all thoses informations in dataframe / array trough the sklearn API?

**Comment by etveritas** [comment]
@jcr-lyxor  you can see https://xlearn-doc.readthedocs.io/en/latest/python_api/index.html#scikit-learn-api-for-xlearn to use sklearn API, but maybe have some small problem, we are improving sklearn API. And this information just print to console when are training.

**Comment by jcr-lyxor** [comment]
Ok I am using it on windows it is working very well. I just want these information. Any idea how I could get them? Are these information written in a file maybe?

**Comment by etveritas** [comment]
@jcr-lyxor You can copy this console information by hand-copy.

**Comment by jcr-lyxor** [comment]
Ok I look forward to have them in the api.
